src/main/java/calculationEngine/battle/CeBattle.java

```diff
@@ -100,15 +100,21 @@ public void flee() { //ToDo: in Progress
         }
     }
 
-    public boolean catchBeast() {
+    public boolean catchBeast(CeItem item) throws Exception {
         System.out.println("Ce_Catch");
         boolean caught = false;
         if (turnPlayer1) {
             turnPlayer1 = false;
-            // CeItem item = new CeItem(1); // Replace with Inventory Use of Cage
-            caught = CeCatching.isCaught(cePlayer1, selectedFightEntityPlayer2, CeLoot.lootItem("cage")); // Replace with Inventory use of Cage
-            if (caught) setBattleEnd();
-            setActionDone();
+            if(item.getName().equals("Cage")) {
```

Comment:
When the CEItem has a Type you can go into this branch when Type is equal the ENUM.CAGE (see below )